Source/AppAuthCore/OIDServiceDiscovery.h

```diff
@@ -44,6 +44,12 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(nonatomic, readonly) NSURL *authorizationEndpoint;
 
+/*! @brief REQUIRED. URL of the OP's OAuth 2.0 Authorization Endpoint.
+    @remarks device_authorization_endpoint
+    @seealso http://openid.net/specs/openid-connect-core-1_0.html#AuthorizationEndpoint
+ */
```

Comment:
Needs updating. URL is https://tools.ietf.org/html/rfc8628, also this isn't a REQUIRED field.

Reply:
Just updated it